style/style.css

```diff
@@ -0,0 +1,160 @@
+  margin-top: -40px;
+  margin-left: 0;
+  margin-right: 0;
+  cursor: pointer;
+  width: 100%;
+  justify-content: center;
+}
+
+.nav-bar {
+  position: relative;
+  display: flex;
+  flex: 1 1 11.11%;
+  top: 130px;
+  width: 990px;
+  height: 40px;
+  border-radius: 3px;
+  box-shadow: rgba(56, 56, 56, 1) 0 1px 2px;
+  background: linear-gradient(to top, rgb(120, 120, 120) 17px, rgb(100, 100, 100) 17px);
+  align-items: stretch;
+  z-index: 1;
+}
+
+.nav-bar > .btn-item {
+  width: 11.12%;
+  height: 35px;
+  margin: 0 -1px;
+  background-color: transparent;
+  border: none;
+  border-right: 1px solid rgb(66, 66, 66);
+  color: white;
+  font-weight: 400;
+}
+
+.nav-bar > .btn-item:hover {
+  background: linear-gradient(#000, #444, #444, #444, #000);
+}
+
+a {
+  text-decoration: none;
+  color: #08c;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  width: 10%;
+  height: 15px;
+  margin: auto 10px;
+  border-radius: 15px;
+  padding: 3px;
+  color: #fff;
+  font-size: 13px;
+  box-shadow: 0 2px 1px rgba(0, 0, 0, 0.6) inset, rgba(255, 255, 255, 0.2) 0 1px 0;
+}
+
+.search > input {
+  width: 70%;
+  margin-left: 3px;
+  border: none;
+  background-color: transparent;
+}
+
+.main-section {
+  background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 50% 100% no-repeat;
+  background-color: transparent;
+  background-size: 1350px 1315px;
+  width: 100%;
+  height: 550px;
+  text-align: center;
+  position: relative;
+  padding-top: 200px;
+  padding-bottom: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  justify-self: flex-start;
+  color: #fff;
+  font-weight: lighter;
+}
+
+.main-section > .img3 {
+  margin-top: 40px;
+  height: 30px;
+  width: 600px;
+  font-size: 22px;
+  font-weight: lighter;
+}
+
+.main-section > .img3:hover {
+  text-decoration: underline;
+}
+
+.grid {
+  max-width: 1350px;
+  height: 210px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 2px;
+  margin: 2px auto;
+}
+
+.grid > img {
+  width: 100%;
+}
+
+.footer {
+  display: grid;
+  width: 1300px;
+  height: 90px;
+  grid-template-columns: 35% 38%;
+  grid-template-rows: 37% 37%;
+  justify-content: center;
+  align-content: center;
+  font-size: 9px;
+  opacity: 0.7;
+}
+
+.footer-item {
+  text-align: start;
+  align-items: center;
+  justify-content: center;
+  padding-top: 15px;
+}
+
+.footer-item > ul {
+  list-style: none;
+  margin-left: -40px;
+  margin-top: -5px;
+}
+
+.footer-item > ul > li {
+  display: inline;
+  margin: auto 3px;
+}
+
+ul > li > a {
+  padding-right: 10px;
+  padding-left: 10px;
+  border-right: 1px solid gray;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.footer-item > ul > li > img {
+  margin-bottom: -5px;
+}
+
+.footer-item > .span1 {
+  margin-right: 3px;
+  border-right: 1px solid gray;
+}
```

Comment:
- [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and  [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

Kindly use float somewhere in your design. This is to ensure that you gain experience in using float.